assets/sass/_content-section.scss

```diff
@@ -13,6 +13,13 @@
   }
 }
 
+.md {
+  ul,
+  ol {
+      margin-left: 40px;
```

Comment:
this seems like a change that could affect many other pages, no? If not, ignore my comment :)

Reply:
Thanks for the check.  I triple checked all the views/pages that use class "md" and "ul,ol" and they have their own styles, that aren't affected by this change.